build-image/Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN cd /opt/data-producer; \
 # Build SQL Playground Image
 ###############################################################################
 
-FROM flink:1.9.0-scala_2.11
+FROM flink:1.10-scala_2.11
```

Comment:
It's image under here: https://hub.docker.com/layers/flink/library/flink/1.10-scala_2.11/images/sha256-184afd6076aa0d0f3416468be86d5a75635415ccf525ba5c0a2b802100a8ca09?context=explore